client/ayon_core/tools/publisher/publish_report_viewer/report_items.py

```diff
@@ -12,17 +12,9 @@ def __init__(self, plugin_data):
         self.order = plugin_data["order"]
         self.skipped = plugin_data["skipped"]
         self.passed = plugin_data["passed"]
-
-        errored = False
-        for instance_data in plugin_data["instances_data"]:
-            for log_item in instance_data["logs"]:
-                errored = log_item["type"] == "error"
-                if errored:
-                    break
-            if errored:
-                break
-
-        self.errored = errored
+        self.actions = plugin_data["action_items"]
+        self.errored = plugin_data["errored"]
```

Comment:
Do we want to get rid of the "instances_data" loop here ?

Reply:
I think so, but I might be wrong... The information is already available without the loop, and since I was already adding "action_items" and "is_blocking" to the plugin data item, it made sense to me to pass "errored" at the same time instead of retrieving it separately. I don't remember if that was the only reason, but looking at the code, I looks like it.

Reply:
can I resolve this one?